### ISSUE TITLE
And again: ValueError: dictionary update sequence element #0 has length 1; 2 is required

### ISSUE THREAD

**Issue by Shazgul** [Hadesy2k/sqlivulscan#5]
Checking the website whether it's online or not.
Connected, URL is valid.
Traceback (most recent call last):
  File "sqliscan.py", line 214, in <module>
    SqliScan(urls)
  File "sqliscan.py", line 64, in __init__
    self.urlRequest(self.urlReader(urls))
  File "sqliscan.py", line 99, in urlRequest
    self.scanVulnerability(site)
  File "sqliscan.py", line 128, in scanVulnerability
    parms = dict([item.split("=") for item in parsedUrl[4].split("&")])
ValueError: dictionary update sequence element #0 has length 1; 2 is required

**Comment by Hadesy2k** [comment]
it happens when parsedUrl[4].split("&") return string, however I cannot reproduce the error in testing and it returns a list (empty list even if index [4] is empty). Thanks for your report, I would appreciate if you would show me the URL that caused this error.

**Comment by Shazgul** [comment]
For Example:

```
 Website Information
 Domain Name : forum.adguard.com
 Protocol    : https
 Path        : /index.php
 Query[s]    : threads/resolved-http-zonasports-to-channel-php-id-3.14092/

Checking the website whether it's online or not.
Connected, URL is valid.
Traceback (most recent call last):
  File "sqliscan.py", line 214, in <module>
    SqliScan(urls)
  File "sqliscan.py", line 64, in __init__
    self.urlRequest(self.urlReader(urls))
  File "sqliscan.py", line 99, in urlRequest
    self.scanVulnerability(site)
  File "sqliscan.py", line 128, in scanVulnerability
    parms = dict([item.split("=") for item in parsedUrl[4].split("&")])
ValueError: dictionary update sequence element #0 has length 1; 2 is required
```

**Comment by Hadesy2k** [comment]
The bug (not able to catch ValueError) **Line 158** is fixed. Now it should be able to catch the value error.
The url you provided above have no =, so splitting will return only one argument. But dict() require two to create dictionary. Therefore, it raised the ValueError, however the bug is fixed and it will show `Query not found`.
